jenkins/Dockerfile/release/linux-arm64/enterprise/tidb

```diff
@@ -1,7 +1,4 @@
-FROM pingcap/centos-stream:8
-RUN set -e && \
-    dnf install bind-utils curl nmap-ncat -y && \
-    dnf clean all
+FROM pingcap/tidb-base:centos8
```

Comment:
Is all those packages `bind-utils curl nmap-ncat` included in image `pingcap/tidb-base:centos8`

Reply:
yes, the dockerfile is in the PR content